### DISCUSSION TITLE
Got my Second NFT!!!!

### DISCUSSION THREAD

**Discussion by Raghav-Programmer** [Cyfrin/foundry-full-course-cu#338] [General]
I completed lesson-2 and solved the challenge...Can't wait to dive deeper into the Web3 world....Thanks @PatrickAlphaC  for creating this awesome course.....

![#2 NFT](https://github.com/Cyfrin/foundry-full-course-f23/assets/113235744/62f0b293-c777-48a1-a3d6-d1c258714f41)

**Comment by cromewar** [comment]
You are doing great, good Job! Please feel free to ask any questions on this repo.

**Comment by Raghav-Programmer** [comment]
Thanks!!.....sure I love to be a part of this community
